android_configure.py

```diff
@@ -61,6 +61,9 @@ def patch_android():
 elif platform.system() == "Linux":
     host_os = "linux"
     toolchain_path = android_ndk_path + "/toolchains/llvm/prebuilt/linux-x86_64"
+    if platform.machine() in {"x86_64", "x86"}:
+        os.environ['CC_host'] = os.popen("which gcc").read().strip()
+        os.environ['CCX_host'] = os.popen("which g++").read().strip()
```

Comment:
Does the Builtin shutil have something for this?

Reply:
This is a better way, and I will submit it for modification.